apriltags.orogen

```diff
@@ -3,7 +3,11 @@ version "0.1"
 
 using_library "apriltags"
 using_library "frame_helper"
-using_library "opencv"
+begin
+	using_library "opencv"
+rescue
+	using_library "opencv4"
+end
```

Comment:
You can remove that entirely, since `frame_helper` already depends on opencv (and finds the correct version using CMake)

Reply:
Okay, then I will get rid of that block later.